global-components/BaseListLayout.vue

```diff
@@ -67,8 +68,8 @@
         return Vue.component(n) || Pagination
       },
 
-      resovlePostDate(date) {
-        return new Date(date.replace(/\-/g, "/").trim()).toDateString()
+      resolvePostDate(date) {
+        return moment(new Date(date)).format(this.$themeConfig.dateFormat || 'ddd MMM DD YYYY')
```

Comment:
```suggestion
        return moment(date).format(this.$themeConfig.dateFormat || 'ddd MMM DD YYYY')
```

Reply:
> It shows `invalid date` In Safari. Have you considered leveraging `dayjs` instead of `moment`? It's more light-weight and it might resolve the bug.

maybe this change is the cause, I revert it, could you try a grain? It's ok on my FireFox, Chrome, and Edge (sorry for I haven't macOS PC now)

Reply:
Sorry for not noticing your update and comment.

It's still broken. Safari doesn't support this date format, that's why `replace(/\-/g, "/")` exists.

Again, I prefer [dayjs](https://github.com/iamkun/dayjs), and it somehow resolves this problem automatically.